CHANGES.md

```diff
@@ -1,3 +1,31 @@
+## 1.5.3 2018-12-20
+
+Linkerd 1.5.3 includes a Finagle upgrade that fixes some memory related issues and adds support for 
```

Comment:
"fixes some memory related issues" sounds scary. Can we be clearer? "reduces allocation" or "improves GC behavior" or something to that effect